server/storage/interface.go

```diff
@@ -33,17 +33,17 @@ type MetaStore interface {
 	// GetCurrent returns the modification date and data part of the metadata for
 	// the latest version of the given GUN and role.  If there is no data for
 	// the given GUN and role, an error is returned.
-	GetCurrent(gun data.GUN, tufRole data.RoleName) (created *time.Time, data []byte, err error)
+	GetCurrent(gun data.GUN, tufRole data.RoleName, channels ...*Channel) (created *time.Time, data []byte, err error)
```

Comment:
Ahhh, yeah, this is fine. I had a moment of forgetfulness that a `...` can also be left empty by the caller.